packages/rich-text/src/SyncEditorChanges.tsx

```diff
@@ -39,19 +40,18 @@ const useOnValueChanged = (onChange?: (doc: Contentful.Document) => unknown) =>
   React.useEffect(() => {
     const cb = createOnChangeCallback(onChange);
 
-    setEditorOnChange({
-      fn: (document) => {
-        // Skip irrelevant events e.g. mouse selection
-        const operations = editor?.operations.filter((op) => {
-          return op.type !== 'set_selection';
-        });
+    // FIXME double-check this is working
```

Comment:
This should be fine just need to double-check it

Reply:
aaaaand have you double checked? :D

Reply:
I have haha